src/a10-server-side-request-forgery/README.md

```diff
@@ -6,8 +6,10 @@ Run the automated tests for verifying the exercise:
 
 `npm run verify`
 
-Start the server for manual tests:
+Start the server and frontend for manual tests:
 
 `npm start`
 
-Use the Postman collection at the root of the repo for ready to use queries matching this problem
+- Use the frontend at `http://localhost:3006/` and match the queries to the problem.
```

Comment:
would be nice to visualize the image not bytes :)
<img width="1920" alt="Screenshot 2023-10-16 at 12 47 49" src="https://github.com/nearform/owasp-top-ten-workshop/assets/36735501/36d9c280-45d1-455d-bfe7-84f0c4158a46">